syntax/gitcommit.vim

```diff
@@ -30,6 +30,9 @@ else
   syn match gitcommitComment	"^#.*"
 endif
 
+" list extracted from https://www.kernel.org/doc/html/latest/process/submitting-patches.html#when-to-use-acked-by-cc-and-co-developed-by
+syn match   gitcommitPseudoHeader "^\(Signed-off-by\|Acked-by\|Cc\|Co-Developed-by\|Co-authored-by\|Fixes\|Reported-by\|Reviewed-by\|Suggested-by\|Tested-by\): .*$"
```

Comment:
Perhaps be a little more flexible in the final whitespace match after the colon, e.g.:
```suggestion
syn match   gitcommitPseudoHeader "^\(Signed-off-by\|Acked-by\|Cc\|Co-Developed-by\|Co-authored-by\|Fixes\|Reported-by\|Reviewed-by\|Suggested-by\|Tested-by\):\s*.*$"
```

I'd also consider highlighting the left-hand-side (the header name) differently than the value, but that's a matter of aesthetics.

Reply:
Ah yes, I think the whitespace could be more relaxed.

About the tag name highlight, I just did it the same way that tig does it too. I think highlighting the whole line has less visual clutter than only highlighting the tag name.

Reply:
I've changed the whitespace regex in gitsendemail.vim too, and force-pushed the changes.